cinema/src/main/java/davidCinema/cinema/Service.java

```diff
@@ -0,0 +1,35 @@
+package davidCinema.cinema;
+
+
+import java.util.HashMap;
+
+public class Service {
+	
+	static int idCounter = 3;
+	//ArrayList<Movie> arrayList = new ArrayList<Movie>();
+	private HashMap <Integer, Movie> movieMap = new HashMap<>();
+	
+public void setUpMovies() {
+	
+	movieMap.put(1, new Movie("Stargate", "Sci-fi", "A12"));

```

Comment:
Could replace with a call to `addMovie` then you don't have to worry about the id counter being set to a certain number - John L